app/services/batched_remove_status_service.rb

```diff
@@ -40,6 +40,7 @@ def call(statuses)
     # Cannot be batched
     statuses.each do |status|
       unpush_from_public_timelines(status)
+      unpush_from_direct_timelines(status) if status.direct_visibility?
```

Comment:
https://codeclimate.com/github/tootsuite/mastodon/pull/4514

Reply:
And this means... what, exactly?

Reply:
Code Climate says the `call` method is too big. It is a good idea to split the `call` method into several private methods.

Reply:
While this indeed should probably be refactored, it's not completely obvious how, and I don't think this PR makes this particular function any more difficult to understand. So, I would personally say this is out of scope of this Pull Request.